test/unit/security.test.js

```diff
@@ -15,15 +16,15 @@ describe('scope module security', function () {
 
         it('"global" must not be visible', function (done) {
             scope.exec(`
-                expect(typeof global).be('undefined');
-                expect(this.global).be(undefined);
+                expect(typeof global).to.equal('undefined');
+                expect(this.global).to.be.undefined;
```

Comment:
See if this assertion is meant to check for absence, or the value being explicitly `undefined`.